testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/EncodingPersistenceTestCase.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.test.manualmode.management.persistence;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+import jakarta.inject.Inject;
+import org.apache.commons.io.FileUtils;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.dmr.ModelNode;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.ManagementClient;
+import org.wildfly.core.testrunner.ServerControl;
+import org.wildfly.core.testrunner.ServerController;
+import org.wildfly.core.testrunner.WildFlyRunner;
+
+import static org.jboss.as.domain.management.ModelDescriptionConstants.VALUE;
+
+@RunWith(WildFlyRunner.class)
+@ServerControl(manual = true)
+public class EncodingPersistenceTestCase {
+    static final PathAddress PROPERTY_ADDR = PathAddress.pathAddress(ModelDescriptionConstants.SYSTEM_PROPERTY, "test_property");
+    @Inject
+    private ServerController serverController;
+
+    /**
+     * Tests that we can write and read the persistence XML when the server is launched using an encoding different that UTF-8
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testEncodingManagementClient() throws Exception {
+        String originalArgs = System.getProperty("jboss.args");
```

Comment:
This is the problem, I'm getting the jboss.args here :( I forgot to change it when I was developing the test case